testdata/formula_contrasts/SRP254919.contrasts_string.yaml

```diff
@@ -0,0 +1,13 @@
+contrasts:
+  - id: treatment_mCherry_hND6_
+    formula: "~ treatment"
+    comparison: ["treatment", "mCherry", "hND6"]
+  - id: treatment_mCherry_hND6_sample_number
+    formula: "~ treatment + sample_number"
+    comparison: ["treatment", "mCherry", "hND6"]
+    blocking_factors: ["sample_number"]
+  - id: treatment_mCherry_hND6_sample_number_complex
+    formula: "~ 0 + treatment + sample_number"
+    comparison: ["treatment", "mCherry", "hND6"]
+    blocking_factors: ["sample_number"]
+    string: "treatmenthND6 - treatmentmCherry"
```

Comment:
I would prefer something more descriptive here, e.g. `make_contrasts_str`

Reply:
Sure, makes sense, I appreciate your feedback. I've changed this.